src/index.html

```diff
@@ -7,7 +7,14 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars__star">
+      <div class="stars stars--0"></div>
+      <div class="stars stars--1"></div>
+      <div class="stars stars--2"></div>
+      <div class="stars stars--3"></div>
+      <div class="stars stars--4"></div>
+      <div class="stars stars--5"></div>
```

Comment:
Please doublecheck the task requirements:
![image](https://github.com/mate-academy/layout_stars/assets/80700247/3193da78-36ca-43cd-8e08-a292090a8e88)

You have to implement 6 html blocks and each block must include 5 stars

Reply:
new commit added